editor/store/reducer.js

```diff
@@ -559,6 +559,16 @@ export function currentPost( state = {}, action ) {
 			}
 
 			return mapValues( post, getPostRawValue );
+
+		case 'RESET_AUTOSAVE':
```

Comment:
I feel that this surfaces another bug (or not, depends on the interpretation :) ) about autosaves. On draft posts, auto-save update the draft itself, so when you refresh the page you get the auto-saved version which is different from the way it works for published posts.

Maybe that's the intent but in that case maybe we shouldn't consider the post as dirty anymore if it's auto saved when it's a draft. Maybe auto-saving a draft is not an auto-save but just a regular save.

For me, it's fine to address later.

Reply:
>On draft posts, auto-save update the draft itself

Yep, I noticed this too. Specifically, we should be accounting for this server-side logic in the client as well:

https://github.com/WordPress/gutenberg/blob/7141ce69260848cbe9ec5a47f2c51bd078b98a2e/lib/class-wp-rest-autosaves-controller.php#L182-L185

I do think this is a separate issue though.